examples/prompt-navigation.js

```diff
@@ -0,0 +1,111 @@
+
+const Enquirer = require('../index');
+const utils = require('../lib/utils');
+
+const enquirer = new Enquirer({
+  actions: {
+    ctrl: {
+      o: 'previousPrompt'
+    }
+  }
+});
+
+const previousPrompt = async function () {
+  await this.submit();
+  this.emit('previousPrompt');
+};
+
+let currentPromptIndex = 0;
```

Comment:
Is there any reason why this was defined outside the `prompt` closure?

Reply:
No, and it would probably be better inside because then the `prompt` function would be reusable... nice catch!